mandatory/3-playing-computer.js

```diff
@@ -7,36 +7,53 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+  - b is a parameter so is undeclared
   2. Remove the line that throws the error.
   3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  x = 2 and a = 6, d = 4 and then 6 and e = 9 and then 13
```

Comment:
almost, not quite. Just enumerate the numbers printed as the `console.log` calls would